### PR TITLE
#2349 add a spdx compliant license, organization and scm information to the created pom of published mbeddr artifacts

### PR DESCRIPTION
**Pull request by KemalSoysal**
Scanning projects for secure bill of materials (SBOM) is faciliated by the providing SPDX compliant
license, organization and source code information.

The modification on the build scripts enhance the created `pom.xml` by the information.

The users of the platform will benefit from the additional information by using automated processes that analyze the `pom.xml` in their projects.

### CODE REVIEW

**Review by alexanderpann — CHANGES_REQUESTED** [review]
We are currently in the process of relicensing mbeddr based on customer requests. We probably switch to EPL 2.0 without a secondary license. Can you please use the 2.0 version? Thanks.

The alternative would be to not wait for the approval of the change and merge.

**Comment by KemalSoysal** [comment]
> We are currently in the process of relicensing mbeddr based on customer requests. We probably switch to EPL 2.0 without a secondary license. Can you please use the 2.0 version? Thanks.
> 
> Please wait with the merging.

When the decision has been made, a change can be introduced by cherry-picking the commit for `EPL-2.0`.
I will not delete the original branch.

**Review by alexanderpann — APPROVED** [review]

**Pull request merged**
